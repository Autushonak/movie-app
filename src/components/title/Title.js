import styled from 'styled-components';
import PropTypes from 'prop-types';
import { colors } from '../../constants/colors';

const Title = styled.h1`
  text-transform: uppercase;
  color: ${colors.WHITE};
`;

Title.propTypes = {
  children: PropTypes.string,
};

export default Title;
