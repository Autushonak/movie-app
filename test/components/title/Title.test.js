import 'jest-styled-components';
import React from 'react';
import { shallow } from 'enzyme';
import Title from '../../../src/components/title/Title';

describe('Title ', () => {
  test('should be rendered correctly', () => {
    const component = shallow(<Title>Any title</Title>);
    expect(component).toMatchSnapshot();
  });
});
