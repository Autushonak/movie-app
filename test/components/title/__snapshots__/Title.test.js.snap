// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title  should be rendered correctly 1`] = `
.c0 {
  text-transform: uppercase;
  color: #FFFFFF;
}

<h1
  className="c0"
>
  Any title
</h1>
`;
